From 89cd64a6350d40e9586637db3f3025bee2fda6fb Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 13 Jan 2016 14:24:20 +0100
Subject: [PATCH] dracut: use cpio with -R root:root rather than -R 0:0

if a user with name "0" exists, there is a problem, but who would do
such a thing??

Resolves: rhbz#1049763
---
 dracut | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut b/dracut
index adad6f3..9e9e529 100755
--- a/dracut
+++ b/dracut
@@ -333,7 +333,7 @@ if strstr "$modules_loaded" " fips " && command -v prelink >/dev/null; then
 fi
 
 type pigz &>/dev/null && gzip=pigz || gzip=gzip
-if ! ( set -o pipefail; umask 077; cd "$initdir"; find . |cpio -R 0:0 -H newc -o --quiet| \
+if ! ( set -o pipefail; umask 077; cd "$initdir"; find . |cpio -R root:root -H newc -o --quiet| \
     $gzip -9 > "$outfile"; ); then
     derror "dracut: creation of $outfile failed"
     rm -f "$outfile"
